### DISCUSSION TITLE
http-client v6.4.7 breaks e-mail in my Laravel application

### DISCUSSION THREAD

**Discussion by Mr-Chimp** [symfony/symfony#54792] [Q&A]
Apologies if this is not the best place for this, we have a Laravel application that sends e-mail using Mailgun via their API, we use the packages mailer-mailgun and mailer (they are both v6.4.7) and when symfony/http-client gets updated to v6.4.7 e-mail stops sending with this error:

Symfony \ Component \ Mailer \ Exception \ HttpTransportException
Unable to send an email: 400 Bad Request (code 400).

PHP version: 8.2.18
Laravel Version: 10.48.10

I manually downgraded http-client to v6.4.6 and it works without issue. I used `composer require symfony/http-client v6.4.6` to downgrade the package.

Any assistance is appreciated, thanks

**Comment by HypeMC** [comment]
It's hard to know what's wrong without seeing what your email, and therefore your HTTP request, looks like. Could you create a [reproducer](https://symfony.com/doc/current/contributing/code/reproducer.html)?
